kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm-ui/src/build/revapi-config.json

```diff
@@ -22,11 +22,42 @@
       }
     }
   },
-
   "ignores": {
     "revapi": {
-    "_comment": "Changes between 7.48.0.Final and the current branch. These changes are desired and thus ignored.",
-    "ignore": [
+      "_comment": "Changes between 7.63.0.Final and the current branch. These changes are desired and thus ignored.",
+      "ignore": [
+        {
+          "code": "java.annotation.attributeValueChanged",
```

Comment:
from line 30 --> 60 have to be removed regarding is duplicated at the end of "ignore" list

Reply:
it's not duplicated, the second one is related to swagger annotation.

Reply:
@bxf12315 what I mean is in this file we only need the three last blocks you added to ignore list. This first two ones are duplicated and have to be removed.

Reply:
them different 
1  "annotationType": "javax.ws.rs.DefaultValue",
          "annotation": "@javax.ws.rs.DefaultValue(\"#1e90ff\")",
2  "annotationType": "io.swagger.annotations.ApiParam",
          "annotation": "@io.swagger.annotations.ApiParam(value = \"svg active node border color\", required = false, example = \"#1e90ff\")",

Reply:
@bxf12315 Line 50 is equals to L106 and L34 = L122

Reply:
Done